src/interface/blas2/spr.cpp.in

```diff
@@ -36,7 +36,7 @@ template typename SB_Handle::event_t
 _spr<SB_Handle, ${INDEX_TYPE}, ${DATA_TYPE}, ${container_t0}, ${INCREMENT_TYPE},
      ${container_t1}>(SB_Handle& sb_handle, char _Uplo, ${INDEX_TYPE} _N,
                       ${DATA_TYPE} _alpha, ${container_t0} _vx,
-                      ${INCREMENT_TYPE} _incx, ${container_t1} _mPA);
+                      ${INCREMENT_TYPE} _incx, ${container_t1} _mPA, const typename SB_Handle::event_t& _dependencies);
```

Comment:
Thanks for your patch, @muhammad-tanvir-1211.

Just a minor thing, the code format didn't consider the `.cpp.in` files.

```suggestion
                      ${INCREMENT_TYPE} _incx, ${container_t1} _mPA,
                      const typename SB_Handle::event_t& _dependencies);
```

Reply:
quick note on this file. `spr` is the only interface that has template parameter explicitly written. I think we can remove them and it should work fine.